mods/ra/maps/fort-lonestar/rules.yaml

```diff
@@ -53,6 +53,8 @@ World:
 			endless: Endless mode
 		Default: hard
 		DisplayOrder: 5
+	MPStartLocations:
+		SeparateTeamSpawnsCheckboxVisible: false
```

Comment:
I'd lock the values as well (using Locked: true and Enabled: true) on all hidden maps.